app/services/config-persistence/nodes/overlays/video.ts

```diff
@@ -15,18 +15,26 @@ interface IContext {
 }
 
 export class VideoNode extends Node<ISchema, IContext> {
-
   schemaVersion = 1;
 
-
-  save(context: IContext) {
+  async save(context: IContext) {
     const filePath = context.sceneItem.getObsInput().settings['local_file'];
+
+    // Don't bother saving media sources that don't have
+    // a file currently loaded
+    if (!filePath) return;
```

Comment:
What if I want to save filters for the source and add the file a little bit later?

Reply:
We don't support that.  The overlay system is meant for exporting finished overlays.  Anything that won't import in a good way for an end user, like a media source without a file, will be ignored.  The main people exporting these files will be our designers.